src/components/modal/construction-modal.hbs

```diff
@@ -1,7 +1,11 @@
 <div id="construction-modal" class="modal" role="dialog" aria-modal="true" aria-labelledby="modalTitle" aria-describedby="modalDesc">
   <div class="modal__flex">
     <div class="modal__content">
-      <button class="modal__close" aria-label="close modal" data-modal-hide="construction-modal">&times;</button>
+      <button class="modal__close" aria-label="close modal" data-modal-hide="construction-modal">
+        <svg class="icon icon-close">
```

Comment:
Previously, Alison has recommended that we use `.sr-only` text instead of `aria-label` for button text so that they're not 'empty'